### DISCUSSION TITLE
Changing description while importing

### DISCUSSION THREAD

**Discussion by boski1987** [firefly-iii/firefly-iii#6190] [General]
I get my bank data (ING in Poland) thru Nordigen. Unfortunately bank provides data without original transaction description and I end up with every transaction described as "done" which makes whole data hard to comprehend. 
I checked json from Nordigen and every transaction has these value:
` "remittanceInformationUnstructured": "done"`
Is there a way to change it on import for example to creditor name? I tried rules but afaik I can't set there variable for description.

**Comment by JC5** [comment]
I'm afraid not. You can only use the rules to set the description to something neutral, or set it to something based on for example the creditor name, but you must create a rule for each variation.
